### PR TITLE
Add attendance and population scripts

### PR DESCRIPTION
**Pull request by Laura-Dale**
Please check this

### CODE REVIEW

**Review by alice-hannah — CHANGES_REQUESTED** [review]
Hi Laura,

These scripts look great and seem to be mostly working - just a couple of wee bits that need changed and some suggested (but non-essential) changes too. I've added comments on individuals lines - happy for us to go through on a call so you can see how to make the changes on GitHub. 

One general question I had was about the coding of missing values: I know these are already coded as x/z/c in the attendance data, and you do this in the code for the population data - would also having the numeric value columns be useful for making the charts? (Like the attainment data files where we have a value column and a value_label column). Maybe not and we can always add in later if we change our mind but just thought I'd check.

Thanks,
Alice

File: code/03_attendance.R [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    ~ here("data", "school_summary_statistics", 
+           paste0(.x, "_school_summary_statistics.xlsx")) %>%
+      read_xlsx(sheet = .y, col_types = "text") %>%
+      clean_names()
+  ) %>%
+  
+  mutate(seed_code = as.character(seed_code)) %>%
+  
+  # Data from Attendance sheet is for All Stages - recode NAs to All Stages
+  mutate(student_stage = replace_na(student_stage, "All Stages")) %>%
```
Comment (alicebyers5) [inline code/03_attendance.R:60]:
```suggestion
  mutate(stage = replace_na(stage, "All Stages")) %>%
```

File: code/04_population.R [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  #mutate(fte_teacher_numbers = replace_na(fte_teacher_numbers,"z"))  %>%
+  #mutate(roll = replace_na(roll,"z"))  %>%
+  #mutate(ptr = replace_na(ptr,"z"))  %>%
+  #mutate(average_class = replace_na(average_class,"z")) 
+
+  # Recode missing / suppressed values
+  mutate(fte_teacher_numbers = recode_missing_values(fte_teacher_numbers, label = TRUE),
+         roll = recode_missing_values(roll, label = TRUE),
+         average_class = recode_missing_values(average_class, label = TRUE),
+         ptr = recode_missing_values(ptr, label = TRUE)) %>%
```
[the remark this answers is not in the document]
Comment (LauraStott) [reply code/04_population.R]: I think it would be useful to have the numeric columns for the charts :)

**Review by alice-hannah — APPROVED** [review]
Hi Laura, thanks for making those changes. We can add the numeric value columns in another branch as realise the `recode_missing_values` function isn't merged with this code yet. We can discuss at our meeting.

Thanks,
Alice

**Pull request merged**